docs/src/network_analysis/network_properties.md

```diff
@@ -0,0 +1,35 @@
+# [Caching of Network Properties in `ReactionSystems`](@id network_analysis_caching_properties)
+When calling many of the network API functions, Catalyst calculates and caches
+in `rn` a variety of information. For example the first call to
+```julia
+rcs,B = reactioncomplexes(rn)
+```
```

Comment:
Could this be made dynamic by creating a simple `rn` model and then applying these on it? The output doesn't;t really matter, so can add `nothing # hide` at the end of each block.